src/tokens.spec.ts

```diff
@@ -46,6 +46,22 @@ describe("test tokens and token computer", async () => {
         const fungibleTokenIdentifier = "FNG-123456";
         identifier = tokenComputer.extractIdentifierFromExtendedIdentifier(fungibleTokenIdentifier);
         assert.equal(identifier, "FNG-123456");
+
+        const numericTokenTicker = "2024-45a190";
```

Comment:
can we double check if lowercase ticker is accepted on the network?

Reply:
Code is here:
 - https://github.com/multiversx/mx-chain-go/blob/master/vm/systemSmartContracts/esdt.go#L721

Reply:
Indeed, thank you.